nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -240,6 +258,7 @@ define([
                                 class: `${cssBaseClass}__message_container`,
                                 dataElement: 'config-message',
                             }),
+                            buildXsvGeneratorButton(events),
```

Comment:
add the xsvGenerator button to the configure panel. It isn't styled at the moment -- will do that later.